src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const populationAll = [...document.querySelectorAll('.population')].map(
+  (item) => +item.textContent.replaceAll(',', ''),
+);
+
+const totalPopulation = populationAll.reduce(
+  (sum, population) => sum + population,
+  0,
+);
+const averagePopulation = Math.round(totalPopulation / populationAll.length);
```

Comment:
Be cautious of division by zero. If 'populationAll' is empty, 'populationAll.length' will be zero, leading to a division by zero error. Consider adding a check to handle this scenario.